client/src/main/java/com/kritsit/casetracker/client/domain/services/Export.java

```diff
@@ -63,4 +64,34 @@ private PdfPTable createTable(List<String> headers, List<String[]> cells){
         }           
         return table;           
     }
+
+	public void exportCaseToPDF(Case c, File file) {
+		Document document = new Document();
+
+        try {
+		    logger.info("Creating file output");
+	        PdfWriter.getInstance(document, new FileOutputStream(file));
+	        document.open();
+	        Font titleFont = new Font(Font.FontFamily.TIMES_ROMAN, 18, Font.BOLD);
+	        Font normalFont = new Font(Font.FontFamily.TIMES_ROMAN, 12);
+	        
+	        document.add(new Paragraph("Case no. "+c.getNumber(), titleFont));
+	        document.add(new Paragraph("Investigating officer:  "+c.getInvestigatingOfficer().getLastName()+" "+c.getInvestigatingOfficer().getFirstName(), normalFont));
+	        document.add(new Paragraph("Incident date: "+c.getIncident().getDate(), normalFont));
+	        document.add(new Paragraph(c.getDescription(), normalFont));
+	        document.add(new Paragraph("Evidence: ", normalFont));
+	        com.itextpdf.text.List list = new com.itextpdf.text.List();
+	        	for(int i=0; i < c.getEvidence().size(); i++){
+	        		list.add(new ListItem(c.getEvidence().get(i).getDescription()));
+	        	}
+            document.close();
+            logger.info("export to PDF: success");
+        } catch (DocumentException e) {
+            logger.error("Error while exporting to PDF");
```

Comment:
Logger.error() can take the Exception as a parameter. It then prints out the stack trace nicely. So these logging lines can be merged into one